### ISSUE TITLE
Add ability to control tally light

### ISSUE THREAD

**Issue by dominik34** [bitfocus/companion-module-birddog-converters#20]
Don’t know if this is even possible but it‘d be great to control the tally lights of bd converters thru companion.
Some switchers do not support ndi tally so this would be an awesome workaround.

**Comment by bryce-seifert** [comment]
I don't believe this is currently possible in the API. It seems you can change whether TallyMode is on/off, but that only applies to the overall mode–not turning the lights on/off themselves
